### DISCUSSION TITLE
Fritz!OS 8 und Fritz!Dect 440

### DISCUSSION THREAD

**Discussion by MarcusAleri** [SeydX/homebridge-fritz-platform#343] [Q&A]
Ist mit einem Update (für Fritz!OS 8) zu rechnen, so dass die Integration des Fritz!Dect 440 Schalter wieder funktioniert (auch die Button - funktioniert seit Update der Fritz!Box auf Fritz!OS 8 nicht mehr)? Gruß Marcus

**Comment by palasinio** [comment]
Es scheint, als würde FritzOS 8 die Funktion nicht mehr über das API bereitstellen. Ebenso die Nutzung von Weblinks.
Vielleicht hilft es, wenn Featurerequests oder Tickets bei AVM erstellt?
